impl/src/test/java/org/jboss/arquillian/warp/impl/utils/TestShrinkWrapUtils.java

```diff
@@ -33,7 +35,7 @@ public void testJavaArchiveFromClass() throws Throwable {
     }
 
     @Test
-    public void testMultipleUse() throws ClassNotFoundException {
+    public void testMultipleUse() throws ClassNotFoundException, IOException {
```

Comment:
Just change to `Exception` - we don't care what exception is thrown in a @ Test - they are all failure conditions.

Reply:
Done.